model/src/main/kotlin/Project.kt

```diff
@@ -62,6 +62,12 @@ data class Project(
     @JsonInclude(JsonInclude.Include.NON_DEFAULT)
     val authors: SortedSet<String> = sortedSetOf(),
 
+    /**
+     * The list of authors declared for this package.
```

Comment:
This should say "copyright holders" instead of "authors". Also, when changing the docs above, remember to change them accordingly also here.

Reply:
I came up with a description 'copyright holders ort copyright statement', given the point that I don't see me as the developer to decide what is about to be filled in here, beyond that rather vague statement